### PR TITLE
Fix property accessor

### PR DESCRIPTION
**Pull request by user135711**
fixes a reference/value bug. 

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/primitives/change-tokens?branch=pr-en-us-6369&view=aspnetcore-2.1#monitoring-configuration-changes-as-a-service)

### CODE REVIEW

File: aspnetcore/fundamentals/primitives/change-tokens.md [hunk 1]
```diff
@@ -103,15 +103,14 @@ The constructor of the implemented class, `ConfigurationMonitor`, registers a ca
 
 [!code-csharp[](change-tokens/sample/Extensions/ConfigurationMonitor.cs?name=snippet2)]
 
-`config.GetReloadToken()` supplies the token. `InvokeChanged` is the callback method. The `state` in this instance is a string that describes the monitoring state. Two properties are used:
+`config.GetReloadToken()` supplies the token. `InvokeChanged` is the callback method. The `state` in this instance is a reference that is used to access the monitoring state. Two properties are used:
```
Comment (HaoK) [inline aspnetcore/fundamentals/primitives/change-tokens.md]: object that represents the monitoring state?

File: aspnetcore/fundamentals/primitives/change-tokens.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ The constructor of the implemented class, `ConfigurationMonitor`, registers a ca
 
 [!code-csharp[](change-tokens/sample/Extensions/ConfigurationMonitor.cs?name=snippet2)]
 
-`config.GetReloadToken()` supplies the token. `InvokeChanged` is the callback method. The `state` in this instance is a reference that is used to access the monitoring state. Two properties are used:
+`config.GetReloadToken()` supplies the token. `InvokeChanged` is the callback method. The `state` in this instance is a reference to the Iconfiguration monitor instance that is used to access the monitoring state. Two properties are used:
```
Comment (guardrex) [inline aspnetcore/fundamentals/primitives/change-tokens.md]: Use ...

> \`IConfigurationMonitor\`

... with the back-ticks to code fence it.

**Comment by scottaddie** [comment]
@HaoK Please take one last look.

**Comment by scottaddie** [comment]
@user135711 Thank you for your help, once again! It's truly appreciated.

**Pull request merged**